src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -196,79 +199,101 @@ private SplitPane getPaneAndStartSearch(BibEntry entry) {
                                               .or(() -> entry.entry().getField(StandardField.URL)).orElse("");
-                            if (StringUtil.isNullOrEmpty(url)) {
-                                return;
-                            }
-                            try {
-                                JabRefDesktop.openBrowser(url, preferencesService.getFilePreferences());
-                            } catch (IOException ex) {
-                                dialogService.notify(Localization.lang("Unable to open link."));
-                            }
-                        });
-                        vContainer.getChildren().addLast(openWeb);
-                    }
-
-                    hContainer.getChildren().addAll(entryNode, separator, vContainer);
-                    hContainer.getStyleClass().add("entry-container");
-
-                    return hContainer;
-                })
-                .withOnMouseClickedEvent((ee, event) -> {
-                    if (!ee.isLocal()) {
-                        listView.getCheckModel().toggleCheckState(ee);
-                    }
-                })
-                .withPseudoClass(entrySelected, listView::getItemBooleanProperty)
-                .install(listView);
-
-        listView.setSelectionModel(new NoSelectionModel<>());
-    }
+                        if (StringUtil.isNullOrEmpty(url)) {
+                            return;
+                        }
+                        try {
+                            JabRefDesktop.openBrowser(url, preferencesService.getFilePreferences());
+                        } catch (IOException ex) {
+                            dialogService.notify(Localization.lang("Unable to open link."));
+                        }
+                    });
+                    vContainer.getChildren().add(openWeb);
+                }
+
+                hContainer.getChildren().addAll(entryNode, separator, vContainer);
+                hContainer.getStyleClass().add("entry-container");
+
+                return hContainer;
+            })
+            .withOnMouseClickedEvent((ee, event) -> {
+                if (!ee.isLocal()) {
+                    listView.getCheckModel().toggleCheckState(ee);
+                }
+            })
+            .withPseudoClass(entrySelected, listView::getItemBooleanProperty)
+            .install(listView);
+
+    listView.setSelectionModel(new NoSelectionModel<>());
+}
+
+// Add this method to open the duplicate entries window
+private void openDuplicateEntriesWindow(BibEntry entry) {
+    // Implementation to open the duplicate entries window
+    Dialog<ButtonType> dialog = new Dialog<>();
+    dialog.setTitle(Localization.lang("Possible duplicate entries"));
+
+    DialogPane dialogPane = new DialogPane();
+    dialogPane.setContent(new Label("Here you can show the duplicate entries and their comparison details."));
```

Comment:
You forgot localization.

Reply:
Hi, what does that mean?

Reply:
Every message to the user needs to be localized. This means, you have to call the Localization.lang method, to mark it to be looked up in the resource bundle. That way jabref knows that is has to look in the resource bundle for the string in the language, the user configured jabref to use. Also you have to add it to the english localization file.
See https://devdocs.jabref.org/code-howtos/localization.html